odbc_fdw.c

```diff
@@ -414,6 +417,30 @@ odbc_connection(odbcFdwOptions* options, SQLHENV *env, SQLHDBC *dbc)
 	ret = SQLDriverConnect(*dbc, NULL, (SQLCHAR *) conn_str.data, SQL_NTS,
 	                       OutConnStr, 1024, &OutConnStrLen, SQL_DRIVER_COMPLETE);
 	check_return(ret, "Connecting to driver", dbc, SQL_HANDLE_DBC);
+	elog_debug("connection opened >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>");
```

Comment:
I think this can be cleaned up (we can keep the message, but the `>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>` should go away).

Reply:
Same for closing the connection

Reply:
done!